src/features/account/AccountBalanceChart.tsx

```diff
@@ -49,9 +48,9 @@ const AccountBalanceChart = ({
   const processedColors = useMemo(
     () => ({
       primaryBackground: processColor(primaryBackground),
-      primaryText: processColor(primaryText),
+      primaryText: processColor('#8C8C8C'),
```

Comment:
Should we pull this color from the theme? Or is it a one off?

Reply:
Its a one off